nested_dict/implementation.py

```diff
@@ -114,25 +108,21 @@ def to_dict(self, input_dict=None):
         return plain_dict
 
     def __str__(self, indent=None):
-        """
-        string version of self
-        """
+        """Representation of self as a string."""
         import json
         return json.dumps(self.to_dict(), indent=indent)
 
 
-class any_type(object):
+class _any_type(object):
```

Comment:
This is the only functional change within this PR.  It seems like an implementation detail that can be hidden.  If you think it should be exposed as part of the public interface I'll revert this part of the change and add a docstring for this class.